rpc/core/pipe.go

```diff
@@ -51,14 +50,11 @@ var (
 	addrBook         *p2p.AddrBook
 	txIndexer        txindex.TxIndexer
 	consensusReactor *consensus.ConsensusReactor
+	eventBus         *types.EventBus
```

Comment:
is this not an "external, thread safe interface" ?

Reply:
thread safe object?